src/esp/physics/bullet/BulletPhysicsManager.cpp

```diff
@@ -371,12 +371,15 @@ bool BulletPhysicsManager::attachLinkGeometry(
         // cache the visual component for later query
         linkObject->visualAttachments_.emplace_back(
             &visualGeomComponent, visual.m_geometry.m_meshFileName);
-        return true;
+      } else {
+        return false;
       }
+    } else {
+      return false;
     }
   }
 
-  return false;
+  return true;
```

Comment:
As far as I can tell, this change is about supporting either (1) multiple render asset instances per link, and/or (2) no render asset instances for a link.